uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/calendar-ctrl.js

```diff
@@ -326,6 +328,15 @@ define([
     };
 
     /**
+     * Reset the months data for before refresh
+     */
+    this._resetMonths = function () {
```

Comment:
Can you remind me

1. What's inside each `month`
2. When you need to reset whatever it's inside each `month`